its/ruling/src/test/resources/expected/php-S4142.json

```diff
@@ -0,0 +1,14 @@
+{
+'project:PHPExcel_1.7.9_doc/Classes/PHPExcel/Shared/PCLZip/pclzip.lib.php':[
+3980,
```

Comment:
I'm wondering about this one, usage of a "[magic constant](http://php.net/manual/en/language.constants.predefined.php)" `__FILE__`.